gensrc/thrift/PaloBrokerService.thrift

```diff
@@ -126,10 +127,17 @@ struct TBrokerPReadRequest {
     4: required i64 length;
 }
 
+enum TSeekWhence {
```

Comment:
And you should change function seek's response with offset like fseek's response